tests/utils/test_names.py

```diff
@@ -43,6 +43,20 @@ def test_normalize_name():
     assert normalize_name("Dupont, J.-R.") == "dupont j r"
     assert normalize_name("Dupont") == "dupont"
     assert normalize_name("Dupont J.R.") == "dupont j r"
+    assert normalize_name("von und zu Hohenstein, F.") == "hohenstein f"
+    assert normalize_name("von und zu Hohenstein, F.",
+                          drop_common_affixes=False) == "vonundzuhohenstein f"
+    assert normalize_name("Jakub, Ibrahim ibn") == "jakub ibrahim ibn"
+    assert normalize_name("o'Neill, Jack") == "neill jack"
```

Comment:
Hmm, do you think it is in fact always a good idea to drop the affixes? E.g, it is very likely that we will have signatures with "Mac Donald" and "MacDonald". If we normalize like this, then these will end into two different blocks.